scripts/vi.py

```diff
@@ -5,15 +5,15 @@
 prompt = lambda content: f'''
 You are a translator for the Vietnamese translation team. You are tasked with translating the following text into Vietnamese. You must follow these instructions:
 - Translate the text into Vietnamese, while keeping the original formatting (either Markdown, MDX or HTML)
-- Inside code blocks, translate the comments but leave the code as-is ; If the code block contains quite plain texts, you MUST provide the translation in <details> tag.
+- Inside code blocks, translate the comments but leave the code as-is ; If the code block contains quite plain texts, you MUST provide the translation in <details> tag
 - Do not translate inline code, the URLs and file paths
-- If the term is abbreviated, keep the original term and provide the translation in parentheses for the first time it appears in the text.
-- If there are any slag or funny joke in english, keep it (do not translate) and give an explanation so vietnamese reader can understand.
-- Use "ta", "chúng ta", "chúng mình", "các bạn" as pronouns.
+- If the term is abbreviated, keep the original term and provide the translation in parentheses for the first time it appears in the text
+- If there are any slag or funny joke in english, keep it (do not translate) and give an explanation so vietnamese reader can understand
+- Use "ta", "chúng ta", "chúng mình", "các bạn" as pronouns
 
 KEEP THESE TERMS (DO NOT TRANSLATE, do NOT add translation in parentheses): model, API, SDK, CLI, HTML, GGUF, AI, training, inference, server, client, notebook, python, Hugging Face, transformers, diffusion, diffuser, data, function, LangGraph, LangChain, Llama, Gemma, token, Unit, pretrain, Live (live stream), form, format, certificate, Space, CodeAgent
```

Comment:
(Just noting here so I don't forget)

Ref our discussion today, we may remove some words from this list because this list is actually based on writing style from my personal blog.